src/config.rs

```diff
@@ -2303,7 +2303,7 @@ pub struct TiKvConfig {
     pub coprocessor: CopConfig,
 
     #[config(skip)]
-    pub coprocessor_v2: CoprocessorV2Config,
+    pub raw_coprocessor: RawCoprocessorConfig,
```

Comment:
It seems to me that renaming from `coprocessor_v2` to `raw_coprocessor` should not be included in this PR. It's another topic and I don't know whether it's appropriate (as https://github.com/tikv/tikv/pull/10229 has adopted the new kvproto and didn't modify the module name...)

Reply:
ye... I see that changes merged during the process. perhaps I should rebase away my own renaming part? I wonder whether @andylokandy could kindly have a check on the changes here. :)

Reply:
I prefer to revert the renamings here. And you can create a new PR to do the renaming.

Reply:
`copr_v2` is the name of the coprocessor framework while `raw_coprocessor` is the request that `copr_v2` can handle. Ideally, `copr_v2` should be invisible to TiKV users. Therefore, I'd like to revert the change here.